databricks_cli/pipelines/cli.py

```diff
@@ -165,6 +165,25 @@ def get_cli(api_client, pipeline_id):
     click.echo(pretty_format(PipelinesApi(api_client).get(pipeline_id)))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Gets a delta pipeline\'s current spec and status')
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def list_cli(api_client):
+    pipelines_api = PipelinesApi(api_client)
+
+    response = pipelines_api.list()
+    pipelines = response["statuses"]
+
+    while "next_page_token" in response["pagination"]:
+        response = pipelines_api.list(page_token=response["pagination"]["next_page_token"])
+        pipelines.extend(response["statuses"])
```

Comment:
Shouldn't this logic should go inside api.py, not cli.py?

Reply:
I'd say cli.py is for understanding and validating the user request, and api.py is for the business logic. For example, when deploying, we validate the parameters in cli.py, but api.py actually uploads local files to DBFS and updates the spec before calling our deploy endpoints.

Reply:
Moved this into api.py.